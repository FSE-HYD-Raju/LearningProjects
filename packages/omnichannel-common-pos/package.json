{
	"name": "omnichannel-common-pos",
	"version": "111.2.0",
	"description": "Common actions, stores and components for omnichannel",
	"main": "lib/index",
	"types": "lib/index.d.ts",
	"oc:es6": "src/index",
	"scripts": {
		"test": "../../testSinglePackage.sh $(pwd)",
		"test-watch": "jest --watch",
		"lint": "eslint tests src --max-warnings=0 ",
		"test-ci": "jest",
		"lint-fix": "eslint src tests --fix",
		"flow": "flow; test $? -eq 0 -o $? -eq 2",
		"clean": "rm -rf lib",
		"build-ts": "tsc --pretty",
		"copy-assets": "copyfiles --up 1 'src/**/*.@(scss|jpg|jpeg|gif|png|svg)' ./lib",
		"build": "cross-env NODE_ENV=production BABEL_ENV=production BABEL_DISABLE_CACHE=1 babel -s inline -d lib/ src/ && npm run build-ts && npm run copy-assets && rimraf -rf messages",
		"watch": "cross-env NODE_ENV=production BABEL_ENV=production BABEL_DISABLE_CACHE=1 babel -w -s inline -d lib/ src/ && rimraf -rf messages",
		"prepublish": "npm run clean && npm run build"
	},
	"jest": {
		"testRegex": "(/tests/.*)\\.test\\.js$",
		"coverageDirectory": "jestcoverage",
		"coverageReporters": [
			"lcov",
			"html"
		],
		"setupFiles": [
			"../../jest-setup.js"
		]
	},
	"author": "",
	"license": "ISC",
	"publishConfig": {
		"registry": "https://artifactory.qvantel.net/artifactory/api/npm/npm-local"
	},
	"dependencies": {
		"alt": "0.18.6",
		"alt-utils": "2.0.0",
		"autosuggest-highlight": "2.1.1",
		"axios": "0.18.0",
		"babel-cli": "6.26.0",
		"babel-core": "6.26.0",
		"babel-eslint": "7.2.3",
		"babel-jest": "23.6.0",
		"babel-loader": "7.1.3",
		"babel-plugin-add-module-exports": "0.2.1",
		"babel-plugin-module-resolver": "3.1.0",
		"babel-plugin-react-intl": "2.4.0",
		"babel-plugin-react-transform": "2.0.2",
		"babel-plugin-transform-class-properties": "6.24.1",
		"babel-plugin-transform-decorators-legacy": "1.3.4",
		"babel-plugin-transform-es2015-classes": "6.24.1",
		"babel-plugin-transform-proto-to-assign": "6.26.0",
		"babel-plugin-transform-react-constant-elements": "6.23.0",
		"babel-plugin-transform-react-inline-elements": "6.22.0",
		"babel-plugin-transform-react-pure-class-to-function": "1.0.1",
		"babel-plugin-transform-react-remove-prop-types": "0.4.20",
		"babel-plugin-transform-runtime": "6.23.0",
		"babel-plugin-typecheck": "3.9.0",
		"babel-polyfill": "6.26.0",
		"babel-preset-es2015": "6.24.1",
		"babel-preset-es2015-loose": "7.0.0",
		"babel-preset-react": "6.24.1",
		"babel-preset-stage-0": "6.24.1",
		"babel-register": "6.26.0",
		"babel-runtime": "6.26.0",
		"bootstrap-sass": "3.3.7",
		"byte-size": "4.0.4",
		"classnames": "2.2.6",
		"connected-react-router": "4.3.0",
		"copyfiles": "2.0.0",
		"country-data": "0.0.31",
		"cross-env": "5.1.4",
		"crypto-js": "3.1.9-1",
		"css-ns": "1.2.1",
		"debug": "4.1.0",
		"diff": "3.4.0",
		"history": "4.7.2",
		"jwt-decode": "2.2.0",
		"langs": "2.0.0",
		"lodash": "4.17.11",
		"moment": "2.21.0",
		"moment-duration-format": "2.2.2",
		"object-assign": "4.1.1",
		"omnichannel-flow-pos": "111.2.0",
		"path": "0.12.7",
		"path-to-regexp": "2.3.0",
		"prop-types": "15.6.1",
		"ramda": "0.24.1",
		"rc-progress": "2.2.5",
		"rc-slider": "8.6.1",
		"react": "16.10.2",
		"react-autosuggest": "9.3.4",
		"react-bootstrap": "0.32.1",
		"react-collapse": "4.0.3",
		"react-confirm": "0.1.14",
		"react-dates": "20.1.0",
		"react-dom": "16.10.2",
		"react-dropzone": "3.13.4",
		"react-flatpickr": "3.6.4",
		"react-formal": "0.25.4",
		"react-formal-inputs": "2.1.1",
		"react-helmet": "5.2.0",
		"react-highlight-words": "0.11.0",
		"react-input-range": "1.3.0",
		"react-intl": "2.7.0",
		"react-motion": "0.5.2",
		"react-motion-ui-pack": "0.10.3",
		"react-onclickoutside": "6.7.1",
		"react-redux": "5.0.7",
		"react-router": "4.2.0",
		"react-router-dom": "4.2.2",
		"react-select": "1.2.1",
		"react-toastr": "3.0.0",
		"react-widgets": "3.5.0",
		"recompose": "0.30.0",
		"redux": "4.0.0",
		"redux-saga": "0.16.0",
		"redux-saga-tester": "1.0.379",
		"shallowequal": "0.2.2",
		"twix": "1.2.1",
		"universal-cookie": "2.1.2",
		"uuid": "3.2.1",
		"weasel-styles": "2.3.1",
		"yup": "0.21.3"
	},
	"devDependencies": {
		"@types/alt": "0.16.35",
		"@types/classnames": "2.2.6",
		"@types/jest": "23.3.9",
		"@types/lodash": "4.14.118",
		"@types/moment-duration-format": "2.2.2",
		"@types/node": "8.0.0",
		"@types/rc-slider": "8.2.3",
		"@types/react": "16.7.18",
		"@types/react-bootstrap": "0.32.14",
		"@types/react-collapse": "4.0.2",
		"@types/react-confirm": "0.1.2",
		"@types/react-dates": "17.1.5",
		"@types/react-dom": "16.0.5",
		"@types/react-flatpickr": "3.7.0",
		"@types/react-helmet": "5.0.7",
		"@types/react-highlight-words": "0.12.0",
		"@types/react-intl": "2.3.9",
		"@types/react-motion": "0.0.27",
		"@types/react-motion-ui-pack": "0.10.1",
		"@types/react-onclickoutside": "6.0.4",
		"@types/react-redux": "6.0.0",
		"@types/react-router": "4.0.25",
		"@types/react-select": "1.2.0",
		"@types/react-toastr": "3.0.3",
		"@types/recompose": "0.26.4",
		"@types/redux-actions": "2.3.0",
		"@types/redux-devtools": "3.0.44",
		"@types/redux-devtools-dock-monitor": "1.1.32",
		"@types/redux-devtools-log-monitor": "1.0.33",
		"@types/redux-logger": "3.0.6",
		"@types/seamless-immutable": "7.1.2",
		"@types/universal-cookie": "2.2.0",
		"@types/webpack": "4.4.0",
		"@types/webpack-env": "1.13.6",
		"@types/yup": "0.26.0",
		"babel-plugin-__coverage__": "11.0.0",
		"babel-plugin-transform-flow-strip-types": "6.22.0",
		"babel-plugin-transform-runtime": "6.23.0",
		"enzyme": "3.7.0",
		"enzyme-adapter-react-16": "1.10.0",
		"enzyme-to-json": "3.3.3",
		"eslint": "4.10.0",
		"eslint-config-omnichannel": "2.2.4",
		"eslint-config-prettier": "1.7.0",
		"eslint-config-react-app": "2.1.0",
		"eslint-loader": "1.9.0",
		"eslint-plugin-babel": "4.1.2",
		"eslint-plugin-flowtype": "2.39.1",
		"eslint-plugin-import": "2.8.0",
		"eslint-plugin-jsx-a11y": "5.1.1",
		"eslint-plugin-prettier": "2.6.0",
		"eslint-plugin-react": "7.4.0",
		"eslint-plugin-react-intl": "1.1.1",
		"flow-bin": "0.70.0",
		"jest": "23.6.0",
		"jest-junit": "3.6.0",
		"lint-staged": "4.3.0",
		"moxios": "0.4.0",
		"pre-commit": "1.2.2",
		"prettier": "1.6.1",
		"react-router-enzyme-context": "1.2.0",
		"react-test-renderer": "16.7.0",
		"redux-devtools": "3.4.1",
		"redux-devtools-dock-monitor": "1.1.3",
		"redux-devtools-log-monitor": "1.4.0",
		"redux-logger": "3.0.6",
		"rimraf": "2.6.2",
		"sinon": "4.3.0",
		"ts-jest": "23.10.5",
		"ts-loader": "5.3.1",
		"typescript": "3.1.6",
		"webpack": "4.26.1"
	},
	"commitInfo": "omni-ui-common-pos|184f176|1571059113|Venkata_Gangadhar_Kodela|develop"
}
