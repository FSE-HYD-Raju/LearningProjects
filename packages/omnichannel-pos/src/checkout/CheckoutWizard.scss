.CheckoutWizard {

    &-step {
        background: theme-color("primary");
        display: flex;
        align-items: flex-start;
		flex: 1 0 0;
		margin-left: 2px;
		&:first-child {
			margin-left: 0;
		}
    }

	&-incomplete-step {
		pointer-events: none;
        background: gray("300");
	}

    &-active {
        background: #3E86B9;
    }

	&-number-container {
		display: flex;
		align-items: center;
	}

    &-number {
        font-weight: $font-weight-bold;
        padding-left: 12px;
        //font-family: OpenSans-Semibold;
        font-size: 50px;
        color: #FFFFFF;
        min-height: 80px;
        display: flex;
        align-items: center;
    }

    &-body {
        display: flex;
        flex-direction: column;
        justify-content: center;
        flex: 1;
        align-items: flex-start;
		min-height: 80px;
		padding: 12px;

    }

    &-label {
        text-transform: uppercase;
        /* Customer details: */
        //font-family: OpenSans-Semibold;
        font-weight: $font-weight-bold;
        font-size: 18px;
        color: #FFFFFF;
        letter-spacing: -0.36px;
    }

    &-type {
        opacity: 0.8;
        font-weight: 400;
        font-size: 14px;
        color: #FFFFFF;
        letter-spacing: 0;
    }

    &-description {
        padding: 0 20px 0 16px;
        >p {
            margin: 0;
        }
    }

    &-steps {
        display: flex;
		align-items: stretch;
		margin-bottom: 0.5rem;
    }

    &-toggle {
        display: flex;
        justify-content: center;
    }

	&-one {
		&:after {
			content: '1';
		}
	}

	&-two {
		&:after {
			content: '2';
		}
	}

	&-three {
		&:after {
			content: '3';
		}
	}

	&-four {
		&:after {
			content: '4';
		}
	}
}
