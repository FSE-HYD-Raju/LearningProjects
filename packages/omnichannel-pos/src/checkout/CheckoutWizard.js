// @flow
import { Component } from "react";
import { cssns, FormattedMessage } from "omnichannel-common-pos";
import PropTypes from "prop-types";
import messages from "../posMessages";
import classnames from "classnames";
const { React } = cssns("CheckoutWizard");

type Props = {
	saveCheckoutConfigurationToBasket: Function,
	setupAvailable?: boolean,
	deliveryAvailable?: boolean,
	summaryAvailable?: boolean,
	paymentAvailable?: boolean,
	clientWidth?: string
};

export default class CheckoutWizard extends Component<Props> {
	static propTypes = {
		saveCheckoutConfigurationToBasket: PropTypes.func,
		setupAvailable: PropTypes.bool,
		deliveryAvailable: PropTypes.bool,
		summaryAvailable: PropTypes.bool,
		paymentAvailable: PropTypes.bool,
		clientWidth: PropTypes.string
	};

	toSetup = () => {
		this.props.saveCheckoutConfigurationToBasket(
			"/servicedesk/checkout/setup",
			"SETUP"
		);
	};

	toDelivery = () => {
		this.props.saveCheckoutConfigurationToBasket(
			"/servicedesk/checkout/delivery",
			"DELIVERY"
		);
	};

	toSummary = () => {
		this.props.saveCheckoutConfigurationToBasket(
			"/servicedesk/checkout/summary",
			"SUMMARY"
		);
	};

	toPayment = () => {
		this.props.saveCheckoutConfigurationToBasket(
			"/servicedesk/checkout/payment",
			"PAYMENT"
		);
	};

	render() {
		const mobileStyles =
			this.props.clientWidth === "tablet"
				? { fontSize: "80%" }
				: { fontSize: "100%" };

		const mobileNumberSize =
			this.props.clientWidth === "tablet"
				? { fontSize: "26px" }
				: { fontSize: "52px" };

		return (
			<div className="steps">
				<div
					onClick={this.toSetup}
					className={classnames({
						step: true,
						"incomplete-step": !this.props.setupAvailable
					})}
				>
					<div className="number-container">
						<span className="number one" style={mobileNumberSize} />
					</div>
					<div className="body">
						<span className="label" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepConfiguration}
							/>
						</span>
						<span className="type" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepConfigurationMessage}
							/>
						</span>
					</div>
				</div>

				<div
					onClick={this.toDelivery}
					className={classnames({
						step: true,
						"incomplete-step": !this.props.deliveryAvailable
					})}
				>
					<div className="number-container">
						<span className="number two" style={mobileNumberSize} />
					</div>
					<div className="body">
						<span className="label" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepDelivery}
							/>
						</span>
						<span className="type" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepDeliveryMessage}
							/>
						</span>
					</div>
				</div>

				<div
					onClick={this.toSummary}
					className={classnames({
						step: true,
						"incomplete-step": !this.props.summaryAvailable
					})}
				>
					<div className="number-container">
						<span
							className="number three"
							style={mobileNumberSize}
						/>
					</div>
					<div className="body">
						<span className="label" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepSummary}
							/>
						</span>
						<span className="type" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepSummaryMessage}
							/>
						</span>
					</div>
				</div>

				<div
					onClick={this.toPayment}
					className={classnames({
						step: true,
						"incomplete-step": !this.props.paymentAvailable
					})}
				>
					<div className="number-container">
						<span
							className="number four"
							style={mobileNumberSize}
						/>
					</div>
					<div className="body">
						<span className="label" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepPayment}
							/>
						</span>
						<span className="type" style={mobileStyles}>
							<FormattedMessage
								{...messages.checkoutStepPaymentMessage}
							/>
						</span>
					</div>
				</div>
			</div>
		);
	}
}
