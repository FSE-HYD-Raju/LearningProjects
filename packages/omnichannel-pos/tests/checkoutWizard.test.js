import React from "react";
import toJson from "enzyme-to-json";

import { mountWithContext, shallowWithContext } from "omnichannel-common-pos";

import CheckoutWizard from "../src/checkout/CheckoutWizard";

describe("CheckoutWizard", () => {
	const context = {
		flux: {
			actions: {
				BasketActions: {
					getActiveBasket: userId => {
						console.log(
							"MOCKED getActiveBasket(), userId:",
							userId
						);
					},
					getActiveBasketByBasketId: activeBasketId => {
						console.log(
							"MOCKED getActiveBasketByBasketId(), activeBasketId:",
							activeBasketId
						);
					}
				}
			}
		}
	};

	/* the component would render without user, but then sessionStorage is required.
     */
	const props = {
		BasketStore: {},
		UserStore: {
			user: {
				id: 1
			}
		},
		PaymentStore: {}
	};

	it("succeeds at shallow mount without props", () => {
		const wrapper = shallowWithContext(<CheckoutWizard />);
		expect(toJson(wrapper)).toMatchSnapshot();
	});

	it("succeeds at deep mount without props", () => {
		mountWithContext(<CheckoutWizard />);
	});

	it("renders as null with minimum props", () => {
		const wrapper = mountWithContext(<CheckoutWizard {...props} />, {
			// attachTo: document.body,
			context
		});

		const checkoutWizard_steps = wrapper.find(".CheckoutWizard-steps");
		expect(checkoutWizard_steps.length).toEqual(
			1,
			".CheckoutWizard-steps not found or there were more than one"
		);

		const steps = checkoutWizard_steps.find(".CheckoutWizard-step");
		expect(steps.length).toEqual(
			4,
			"There should be 4 .CheckoutWizard-step elements, found " +
				steps.length
		);
	});

	it("should initially render all steps as incomplete", () => {
		const wrapper = mountWithContext(<CheckoutWizard {...props} />, {
			// attachTo: document.body,
			context
		});

		const checkoutWizard_steps = wrapper.find(".CheckoutWizard-steps");
		expect(checkoutWizard_steps.length).toEqual(
			1,
			".CheckoutWizard-steps not found or there were more than one"
		);

		const steps = checkoutWizard_steps.find(".CheckoutWizard-step");
		expect(steps.length).toEqual(
			4,
			"There should be 4 .CheckoutWizard-step elements, found " +
				steps.length
		);

		steps.forEach(n => {
			expect(n.hasClass("CheckoutWizard-incomplete-step")).toEqual(
				true,
				"Every .CheckoutWizard-step should also have class CheckoutWizard-incomplete-step"
			);
		});
	});

	describe("step", () => {
		const stepData = [
			{
				name: "one",
				label: "Configuration",
				type: "Configure your services"
			},
			{
				name: "two",
				label: "Delivery",
				type: "Select your delivery methods"
			},
			{
				name: "three",
				label: "Summary",
				type: "Review your order and accept the related contract"
			},
			{
				name: "four",
				label: "Payment",
				type: "Pay for your order"
			}
		];
		let wrapper = null;
		let checkoutWizard_steps = null;
		let steps = null;
		let numberContainers = null;
		function chekcStepDataRenderedCorrectrly(
			numberContainers,
			stepData,
			wrapper,
			idx
		) {
			const numberContainer = numberContainers
				.find(".CheckoutWizard-" + stepData[idx].name)
				.parent();
			expect(numberContainer.length).toEqual(
				1,
				".CheckoutWizard-" +
					stepData[idx].name +
					" not found or there were more than one"
			);

			const body = wrapper.find(".CheckoutWizard-body").at(idx);

			const label = wrapper.find(".CheckoutWizard-label").at(idx);

			expect(label.text().toLowerCase()).toEqual(
				stepData[idx].label.toLowerCase()
			);

			const type = body.find(".CheckoutWizard-type");
			expect(type.length).toEqual(
				1,
				".CheckoutWizard-type not found or there were more than one"
			);
			expect(type.text().toLowerCase()).toEqual(
				stepData[idx].type.toLowerCase()
			);
		}

		beforeEach(() => {
			wrapper = mountWithContext(<CheckoutWizard {...props} />, {
				// attachTo: document.body,
				context
			});

			checkoutWizard_steps = wrapper.find(".CheckoutWizard-steps");
			expect(checkoutWizard_steps.length).toEqual(
				1,
				".CheckoutWizard-steps not found or there were more than one"
			);

			steps = checkoutWizard_steps.find(".CheckoutWizard-step");
			expect(steps.length).toBeGreaterThan(
				0,
				"There should be more than zero .CheckoutWizard-step elements"
			);

			numberContainers = steps.find(".CheckoutWizard-number-container");
			expect(numberContainers.length).toBeGreaterThan(
				0,
				"There should be more than zero .CheckoutWizard-number-container elements"
			);
		});

		it("CheckoutWizard-body length equal stepData.length", () => {
			const body = wrapper.find(".CheckoutWizard-body");
			expect(body.length).toEqual(
				stepData.length,
				".CheckoutWizard-body not found or there were more than one"
			);
		});
		it("CheckoutWizard-label length equal stepData.length", () => {
			const label = wrapper.find(".CheckoutWizard-label");
			expect(label.length).toEqual(
				stepData.length,
				".CheckoutWizard-label not found or there were more than one"
			);
		});

		it("stepData name=one is rendered correctly", () => {
			chekcStepDataRenderedCorrectrly(
				numberContainers,
				stepData,
				wrapper,
				0
			);
		});

		it("stepData name=two is rendered correctly", () => {
			chekcStepDataRenderedCorrectrly(
				numberContainers,
				stepData,
				wrapper,
				1
			);
		});

		it("stepData name=three is rendered correctly", () => {
			chekcStepDataRenderedCorrectrly(
				numberContainers,
				stepData,
				wrapper,
				2
			);
		});

		it("stepData name=four is rendered correctly", () => {
			chekcStepDataRenderedCorrectrly(
				numberContainers,
				stepData,
				wrapper,
				3
			);
		});
	});

	describe("calls given saveCheckoutConfigurationToBasket() with correct arguments", () => {
		const stepData = [
			{
				numeralName: "one",
				name: "SETUP",
				url: "/servicedesk/checkout/setup"
			},
			{
				numeralName: "two",
				name: "DELIVERY",
				url: "/servicedesk/checkout/delivery"
			},
			{
				numeralName: "three",
				name: "SUMMARY",
				url: "/servicedesk/checkout/summary"
			},
			{
				numeralName: "four",
				name: "PAYMENT",
				url: "/servicedesk/checkout/payment"
			}
		];
		let wrapper = null;
		let checkoutWizard_steps = null;
		let steps = null;
		let numberContainers = null;
		let currentStepEntry = null;
		let fnDone = null;

		function saveCheckoutConfigurationToBasket(url, stepName) {
			console.log(
				"MOCKED saveCheckoutConfigurationToBasket(), url, stepName:",
				url,
				stepName
			);
			expect(url).toEqual(currentStepEntry.url);
			expect(stepName).toEqual(currentStepEntry.name);
			fnDone();
		}

		beforeEach(() => {
			wrapper = mountWithContext(
				<CheckoutWizard
					{...props}
					saveCheckoutConfigurationToBasket={
						saveCheckoutConfigurationToBasket
					}
				/>,
				{
					// attachTo: document.body,
					context
				}
			);

			checkoutWizard_steps = wrapper.find(".CheckoutWizard-steps");
			expect(checkoutWizard_steps.length).toEqual(
				1,
				".CheckoutWizard-steps not found or there were more than one"
			);

			steps = checkoutWizard_steps.find(".CheckoutWizard-step");
			expect(steps.length).toBeGreaterThan(
				0,
				"There should be more than zero .CheckoutWizard-step elements"
			);

			numberContainers = steps.find(".CheckoutWizard-number-container");
			expect(numberContainers.length).toBeGreaterThan(
				0,
				"There should be more than zero .CheckoutWizard-number-container elements"
			);
		});

		function numeralNameRenderedCorrectly(done, idx) {
			currentStepEntry = stepData[idx];
			fnDone = done;

			const numberContainer = wrapper.find(
				".CheckoutWizard-" + stepData[idx].numeralName
			);
			expect(numberContainer.length).toEqual(
				1,
				".CheckoutWizard-" +
					stepData[idx].numeralName +
					" not found or there were more than one"
			);
			numberContainer.simulate("click");
		}

		it("stepData numeralName=one is rendered correctly", done => {
			numeralNameRenderedCorrectly(done, 0);
		});
		it("stepData numeralName=two is rendered correctly", done => {
			numeralNameRenderedCorrectly(done, 1);
		});
		it("stepData numeralName=three is rendered correctly", done => {
			numeralNameRenderedCorrectly(done, 2);
		});
		it("stepData numeralName=four is rendered correctly", done => {
			numeralNameRenderedCorrectly(done, 3);
		});
	});
});
