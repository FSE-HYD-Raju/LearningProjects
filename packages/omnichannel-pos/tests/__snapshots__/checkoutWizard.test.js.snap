// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutWizard succeeds at shallow mount without props 1`] = `
<div
  className="CheckoutWizard-steps"
>
  <div
    className="CheckoutWizard-step CheckoutWizard-incomplete-step"
    onClick={[Function]}
  >
    <div
      className="CheckoutWizard-number-container"
    >
      <span
        className="CheckoutWizard-number CheckoutWizard-one"
        style={
          Object {
            "fontSize": "52px",
          }
        }
      />
    </div>
    <div
      className="CheckoutWizard-body"
    >
      <span
        className="CheckoutWizard-label"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Configuration"
          description="Checkout first step: configuration"
          id="checkout-steps-configuration"
          wrapperType="wrapper"
        />
      </span>
      <span
        className="CheckoutWizard-type"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Configure your services"
          description="Checkout configuration message"
          id="checkout-steps-configuration-message"
          wrapperType="wrapper"
        />
      </span>
    </div>
  </div>
  <div
    className="CheckoutWizard-step CheckoutWizard-incomplete-step"
    onClick={[Function]}
  >
    <div
      className="CheckoutWizard-number-container"
    >
      <span
        className="CheckoutWizard-number CheckoutWizard-two"
        style={
          Object {
            "fontSize": "52px",
          }
        }
      />
    </div>
    <div
      className="CheckoutWizard-body"
    >
      <span
        className="CheckoutWizard-label"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Delivery"
          description="Checkout second step: delivery"
          id="checkout-steps-delivery"
          wrapperType="wrapper"
        />
      </span>
      <span
        className="CheckoutWizard-type"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Select your delivery methods"
          description="Checkout delivery message"
          id="checkout-steps-delivery-message"
          wrapperType="wrapper"
        />
      </span>
    </div>
  </div>
  <div
    className="CheckoutWizard-step CheckoutWizard-incomplete-step"
    onClick={[Function]}
  >
    <div
      className="CheckoutWizard-number-container"
    >
      <span
        className="CheckoutWizard-number CheckoutWizard-three"
        style={
          Object {
            "fontSize": "52px",
          }
        }
      />
    </div>
    <div
      className="CheckoutWizard-body"
    >
      <span
        className="CheckoutWizard-label"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Summary"
          description="Checkout second step: summary"
          id="checkout-step-summary"
          wrapperType="wrapper"
        />
      </span>
      <span
        className="CheckoutWizard-type"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Review your order and accept the related contract"
          description="Checkout summary message"
          id="checkout-steps-summary-message"
          wrapperType="wrapper"
        />
      </span>
    </div>
  </div>
  <div
    className="CheckoutWizard-step CheckoutWizard-incomplete-step"
    onClick={[Function]}
  >
    <div
      className="CheckoutWizard-number-container"
    >
      <span
        className="CheckoutWizard-number CheckoutWizard-four"
        style={
          Object {
            "fontSize": "52px",
          }
        }
      />
    </div>
    <div
      className="CheckoutWizard-body"
    >
      <span
        className="CheckoutWizard-label"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Payment"
          description="Checkout second step: payment"
          id="checkout-step-payment"
          wrapperType="wrapper"
        />
      </span>
      <span
        className="CheckoutWizard-type"
        style={
          Object {
            "fontSize": "100%",
          }
        }
      >
        <FormattedMessageWrapper
          defaultMessage="Pay for your order"
          description="Checkout payment message"
          id="checkout-steps-payment-message"
          wrapperType="wrapper"
        />
      </span>
    </div>
  </div>
</div>
`;
